### PR TITLE
12 martin vrachev hw4

### PR DESCRIPTION
**Pull request by mvvrachev**

### CODE REVIEW

File: homework_04/12_Martin_Vrachev/circles.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+        # Touching
+        if a_b in (radii_sum, radii_sub):
+            return RelativePosition.TOUCHING
+
+        # Intersecting
+        if radii_sub < a_b < radii_sum:
+            return RelativePosition.INTERSECTING
+
+        return None
```
Comment (Vicktoria853) [inline homework_04/12_Martin_Vrachev/circles.py:101]: Results are 8.79
You can disable C1001 (old style class), but for unnescessary parens after 'print' keyword I am not sure - in https://github.com/PyCQA/pylint/issues/2801 and https://stackoverflow.com/questions/28694380/pylint-says-unnecessary-parens-after-r-keyword are talking about it.

File: homework_04/12_Martin_Vrachev/test_circles.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+def test_invalid_distance_arguments():
+    '''
+    Tests for invalid araguments
+    '''
+    with pytest.raises(SystemExit) as pytest_wrapped_e:
+        distance(2,3)
+
+    assert pytest_wrapped_e.type == SystemExit
+
+def test_find_realtive_position_invalid_arguments():
```
Comment (Vicktoria853) [inline homework_04/12_Martin_Vrachev/test_circles.py:68]: *relative
/# pylint: disable=invalid-name will help you as I do not see anything wrong with the name.

File: homework_04/12_Martin_Vrachev/test_circles.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    point_1 = Point(2.0, -1.0)
+    point_2 = Point(-2.0, 2.0)
+    assert distance(point_1, point_2) == 5.0
+
+def test_invalid_distance_arguments():
+    '''
+    Tests for invalid araguments
+    '''
+    with pytest.raises(SystemExit) as pytest_wrapped_e:
+        distance(2,3)
```
Comment (Vicktoria853) [inline homework_04/12_Martin_Vrachev/test_circles.py:64]: distance(2, 3) took 0.17 point

**Review by Vicktoria853 — APPROVED** [review]
I think everything is good.

**Review by atanasatanasov03 — APPROVED** [review]
vsichko e super evala

File: homework_04/12_Martin_Vrachev/circles.py (hunk 1 above)
Comment (mvvrachev) [reply homework_04/12_Martin_Vrachev/circles.py:101]: How come the results from pylint are 8.79? When I run it on my machine it says 10 out of 10 plus the checks from the action have passed perfectly. If the result was different from 10, then the checks would have failed.

File: homework_04/12_Martin_Vrachev/test_circles.py (hunk 2 above)
Comment (mvvrachev) [reply homework_04/12_Martin_Vrachev/test_circles.py:68]: Yeah, I will fix the typo, thanks. However I do not get an error from pylint here as well.

**Review by Vicktoria853 — APPROVED** [review]

**Pull request merged**